spec/04-status_api_spec.lua

```diff
@@ -215,6 +276,33 @@ describe("Plugin: prometheus (access via status API)", function()
     assert.matches('kong_datastore_reachable 1', body, nil, true)
   end)
 
+  it("exposes upstream's target health metrics - healthy", function()
+    local res = assert(status_client:send {
+      method  = "GET",
+      path    = "/metrics",
+    })
+    local body = assert.res_status(200, res)
+    assert.matches('kong_upstream_target_health{upstream="mock-upstream",target="' .. helpers.mock_upstream_host .. ':' .. helpers.mock_upstream_port .. '",state="healthy"} 1', body, nil, true)
```

Comment:
let's also check for the correctness of other state (same for other 3 tests)

Reply:
on it!